mm2src/mm2_libp2p/src/atomicdex_behaviour.rs

```diff
@@ -368,6 +373,9 @@ impl AtomicDexBehaviour {
                     error!("Result rx is dropped");
                 }
             },
+            AdexBehaviourCmd::AddPeerAddresses { peer, addresses } => {
+                self.peers_exchange.add_peer_addresses(&peer, addresses);
```

Comment:
I have also noticed a problem - `PeersExchange` behavior keeps track of 100 peers max. So if the node discovers more it can "forget" the addresses and version requests will fail.
https://github.com/KomodoPlatform/atomicDEX-API/blob/version-p2p-request/mm2src/mm2_libp2p/src/peers_exchange.rs#L150

I think it's worth having the `reserved_peers` field in `PeersExchange` and avoid "forgetting" them.